modules/study/src/main/TreeBuilder.scala

```diff
@@ -18,14 +18,19 @@ object TreeBuilder {
         val sit = Game(variant.gameLogic, variant.some, root.fen.some).situation
         sit.playable(false) ?? sit.destinations
       }
-    makeRoot(root, variant).copy(dests = dests.some)
+    val dropsByRole = {
+      val sit = Game(variant.gameLogic, variant.some, root.fen.some).situation
+      sit.playable(false) ?? sit.dropsByRole
+    }
```

Comment:
So this code only adds the drops if the situation is playable. I wonder if all of the places that we use the drops by role should do the same thing? Probably.

Reply:
Maybe in the context of study. I expect that there are other checks made in the case of round